operations/validate.go

```diff
@@ -115,10 +116,15 @@ func validateFile(path string, ac *legacyClient, quiet, includeLong bool, localM
 		LocalModules: localModuleMap,
 		ReadFileFrom: model.ReadFromLocal,
 	}
-	pp, _, err := model.LoadProjectInto(ctx, confFile, opts, "", project)
-	if err != nil {
-		return errors.Wrapf(err, "%s is an invalid configuration", path)
+	if !quiet {
+		opts.UnmarshalStrict = true
+	}
+	pp, validationErrs := loadProjectIntoWithValidation(ctx, confFile, opts, project)
+	grip.Info(validationErrs)
```

Comment:
Did you mean to leave this here?

Reply:
Yes, this is intentionally to communicate any errors with the user.